### PR TITLE
Remove .scripts support

### LINKED ISSUE #678: Remove old `.script` files and `.script` -> `.json` converter

### PR DESCRIPTION
**Pull request by henrykironde**
This also updates the version.py and version.txt files.
It also removes the compile function for .scripts
Removes the parse_script_to_json module.

fixes #678 #679

### CODE REVIEW

File: test/test_integration.py [hunk 1]
```diff
@@ -6,39 +6,219 @@
...
+                            {\n
+                                "dialect": {},\n
+                                "name": "simple_csv",\n
+                                "schema": {},\n
+                                "url": "http://example.com/simple_csv.txt"\n
+                            }\n
+                        ],\n
+                        "retriever": "True",\n
+                        "retriever_minimum_version": "2.0.0-dev",\n
+                        "script_version": 1.0,\n
```
Comment (ethanwhite) [inline test/test_integration.py]: All of these `script_version`'s should be changed to `version` to match the recent update in the scripts.

File: test/test_integration.py [hunk 2]
```diff
@@ -6,39 +6,219 @@
...
+                                            "type": "int"\n
+                                        }\n
+                                    ]\n
+                                },\n
+                                "url": "http://example.com/autopk_csv.txt"\n
+                            }\n
+                        ],\n
+                        "retriever": "True",\n
+                        "retriever_minimum_version": "2.0.0-dev",\n
+                        "script_version": 1.0,\n
```
Comment (ethanwhite) [inline test/test_integration.py]: Update to `version`.

File: test/test_integration.py [hunk 3]
```diff
@@ -6,39 +6,219 @@
...
+                                        "type": "ct-double"\n
+                                    }\n
+                                ]\n
+                            },\n
+                            "url": "http://example.com/crosstab.txt"\n
+                        }\n
+                    ],\n
+                    "retriever": "True",\n
+                    "retriever_minimum_version": "2.0.0-dev",\n
+                    "script_version": 1.0,\n
```
Comment (ethanwhite) [inline test/test_integration.py]: Update to `version`.

File: test/test_integration.py [hunk 4]
```diff
@@ -6,39 +6,219 @@
...
+                                                "type": "ct-double"\n
+                                            }\n
+                                        ]\n
+                                    },\n
+                                    "url": "http://example.com/autopk_crosstab.txt"\n
+                                }\n
+                            ],\n
+                            "retriever": "True",\n
+                            "retriever_minimum_version": "2.0.0-dev",\n
+                            "script_version": 1.0,\n
```
Comment (ethanwhite) [inline test/test_integration.py]: Update to `version`.

File: test/test_integration.py [hunk 5]
```diff
@@ -6,39 +6,219 @@
...
+                                        "type": "int"\n
+                                    }\n
+                                ]\n
+                            },\n
+                            "url": "http://example.com/skip_csv.txt"\n
+                        }\n
+                    ],\n
+                    "retriever": "True",\n
+                    "retriever_minimum_version": "2.0.0-dev",\n
+                    "script_version": 1.0,\n
```
Comment (ethanwhite) [inline test/test_integration.py]: Update to `version`.

File: test/test_integration.py [hunk 6]
```diff
@@ -6,39 +6,219 @@
...
+                            {\n
+                                "dialect": {},\n
+                                "name": "extra_newline",\n
+                                "schema": {},\n
+                                "url": "http://example.com/extra_newline.txt"\n
+                            }\n
+                        ],\n
+                        "retriever": "True",\n
+                        "retriever_minimum_version": "2.0.0-dev",\n
+                        "script_version": 1.0,\n
```
Comment (ethanwhite) [inline test/test_integration.py]: Update to `version`.

**Comment by henrykironde** [comment]
@ethanwhite, I guess we are done here

**Pull request merged**